engine/framework/worker.go

```diff
@@ -197,8 +198,8 @@ func NewBaseWorker(
 		errCenter:        errctx.NewErrCenter(),
 		clock:            clock.New(),
 		userMetaKVClient: kvclient.NewPrefixKVClient(params.UserRawKVClient, ctx.ProjectInfo.UniqueID()),
-		logger:           logutil.NewLogger4Worker(ctx.ProjectInfo, masterID, workerID),
 		metricFactory:    promutil.NewFactory4Worker(ctx.ProjectInfo, MustConvertWorkerType2JobType(tp), masterID, workerID),
+		logger:           engineLog.Logger(frameLog.WithWorkerID(frameLog.WithMasterID(*ctx, masterID), workerID)),
```

Comment:
Here is the cached `logger` of `DefaultBaseMaster`/`DefaultBaseWorker` for:
1. Return value of `Logger()` interface method
2. method which has not context input

Since `context` may come from different scenerios, like grpc(`SubmitJob`), runtime(`Init`), we will may put a logger on context if we need to achieve:
```
[A=xx] ....   => [A=xx][component=xxx] ... => [A=xx][component=xxx][meta=xxx] ...
```  
But not in this pr.